test/mysql-chassis.js

```diff
@@ -316,6 +326,10 @@ describe('mysql-chassis', () => {
 
   // })
 
+  // describe('transaction methods', () => {
+
```

Comment:
TBH, I have no idea how to test this without the actual interaction with the DB. So to answer your question, no plans to update this. I'd be happy to do it however if you or @bradwestfall has an idea how to do it.